### ISSUE TITLE
RealmSwift 10.29.0 increased my WatchKit App bundle size

### ISSUE THREAD

**Issue by veluloganathan** [realm/realm-swift#7964]
### How frequently does the bug occur?

All the time

### Description

I did pod update for realm-swift to run/debug in XCode 14. But when i submit a build to Appstore i got an error as Size Limit Exceeded  the 75MB size limit.

Help me 

### Can you reproduce the bug?

Yes, always

### Version

10.29.0

### What SDK flavour are you using?

Local Database only

### Are you using encryption?

No, not using encryption

### Platform OS and version(s)

iOS 16, Watch OS 8

### Build environment

Xcode version: ...
Dependency manager and version: ...

**Comment by dianaafanador3** [comment]
@veluloganathan from which version are you migration from? Are there other frameworks updates from the last time you uploaded to App Store?

**Comment by veluloganathan** [comment]
i migrated from `10.25.0` and i havn't updated any other frameworks.

**Comment by dianaafanador3** [comment]
Have you tried using a older version of Xcode?, I would try this but I cannot upload anything to the AppStore, this may help us identify if the issue is in our side or Apple side.

**Comment by veluloganathan** [comment]
No, But i managed to install realm-swift using SPM and it solves the issue for now. 

FYI: I do see librealm-monorepo.a added to my built product(watchOS App).

**Comment by dianaafanador3** [comment]
@veluloganathan I made some tests and It look like we haven't made any big changes in our side to represent any big increase in the binary size. 
In Xcode 14 builds for watchOS now include arm64 architecture which would explain the increase in the size, and because build flags for Cocoapods and SPM are different there may be a slight difference between the builds. 
This difference may be the cause for your app binary size to go over the max app size for a watchOS App, because your app may be almost in the limit to go over 75Mb.
